htdocs/user/class/user.class.php

```diff
@@ -2487,6 +2487,56 @@ public function send_password($user, $password = '', $changelater = 0)
 		}
 	}
 
+	// phpcs:disable PEAR.NamingConventions.ValidFunctionName.ScopeNotCamelCaps
```

Comment:
do not use phpcs:disable to introduce new not CamelCaps code

Reply:
Done